driftbase/api/players/__init__.py

```diff
@@ -178,6 +181,12 @@ def get(self, args):
                 # Note! This is a particular optimization in case where player group is empty
                 return []
             query = query.filter(CorePlayer.player_id.in_(player_ids))
+        elif 'player_name' in args:
+            if '*' not in args["player_name"]:
```

Comment:
I suggest grabbing player_name out of args into a local variable since it's used multiple times here